Client/src/Components/Notifications/index.jsx

```diff
@@ -0,0 +1,10 @@
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const Notification = () => {
+  return <ToastContainer />;
+};
+
+export const notify = (message) => toast(message);
```

Comment:
ESlint doesn't like exporting both a component and a function from the same file.  Apparently it will disable fast refresh for the component.  I don't know what fast refresh is or how it works, but it seems a simple fix to avoid disabling it, so we may as well.

Recommend moving this to a separate JS file, maybe `notify.js` to resovle the warning.

```
import { toast } from "react-toastify";
export default (message) => toast(message);
```

should do it.

Reply:
I addressed this issue.